time_series_transformer/decoder.py

```diff
@@ -55,19 +61,23 @@ def __init__(self,
         elif chunk_mode is None:
             MHA = MultiHeadAttention
         else:
+            # pylint: disable=line-too-long
             raise NameError(
                 f'chunk_mode "{chunk_mode}" not understood. Must be one of {", ".join(chunk_mode_modules.keys())} or None.')
 
-        self._selfAttention = MHA(d_model, q, v, h, attention_size=attention_size)
-        self._encoderDecoderAttention = MHA(d_model, q, v, h, attention_size=attention_size)
+        self._selfAttention = MHA(
+            d_model, q, v, h, attention_size=attention_size)
+        self._encoderDecoderAttention = MHA(
+            d_model, q, v, h, attention_size=attention_size)
         self._feedForward = PositionwiseFeedForward(d_model)
 
         self._layerNorm1 = nn.LayerNorm(d_model)
         self._layerNorm2 = nn.LayerNorm(d_model)
         self._layerNorm3 = nn.LayerNorm(d_model)
 
-        self._dopout = nn.Dropout(p=dropout)
+        self._dropout = nn.Dropout(p=dropout)
```

Comment:
Haha nicely spotted, I would never had seen it !

Reply:
Nay, you would have seen it if you had a huge 4k monitor like mine.

Reply:
Haha *cries in 15' screen*